### DISCUSSION TITLE
Window "Are you sure?" doesn't work well

### DISCUSSION THREAD

**Discussion by Pablo-163** [dbeaver/dbeaver#36265] [Q&A] [answered]
### Description

When I click on close button I get "Are you sure?" window. If I really want to close - all ok. I press "Yes" and close dbeaver.
But if I missclicked and don't want to close dbeaver - situation is the same: "No" button also close dbeaver. 

[Запись экрана от 15.11.2024 17:14:35.webm](https://github.com/user-attachments/assets/a723c568-6811-48fd-b68e-33baf2c4aeb1)

### DBeaver Version

Community Edition 24.1.1

### Operating System

Ubuntu 22.04.5 LTS

**Comment by mthread1** [comment, marked as answer]
This was fixed in newly versions.
Try to install update and check it again.

**Comment by Pablo-163** [comment]
Yes!  Version 24.2.5 work well with that window.
